### DISCUSSION TITLE
X.com, LinkedIn etc are broken

### DISCUSSION THREAD

**Discussion by gerroon** [uBlockOrigin/uAssets#25410] [Q&A] [answered]
Hi

It seems like there has been some kind of update in UblockO or in the blocklist, I cant login to any of these while the extension is on in Firefox (win 11, 129.02).

Disabling UblockO lets me login so it seems related. I already disabled Firefox Enhanced tracking for these sites.

![image](https://github.com/user-attachments/assets/7786b061-b3ce-4dab-9428-0325cb8071c0)

![image](https://github.com/user-attachments/assets/3e75a732-32ae-4ea2-93bb-56ff9185135e)

**Comment by gorhill** [comment]
Post here troubleshooting information: Go to the webpage with the issue > Click the [uBO toolbar icon](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface) > Click the [chat icon 💬](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > Click "Troubleshooting Information" > Report content here (https://www.reddit.com/link/173jmog/video/bogwf92af9tb1/player)

**Reply by gerroon** [reply]
while Linked in open, see the image please, This is a recent issue it seems, it has been working fine until last week I think.
```
uBlock Origin: 1.59.1rc7
Firefox: 129
filterset (summary):
 network: 196417
 cosmetic: 269121
 scriptlet: 53691
 html: 2248
listset (total-discarded, last-updated):
 added:
  https://filters.adtidy.org/extension/ublock/filters/14.txt: 82485-155, 4d.13h.6m
  https://malware-filter.gitlab.io/pup-filter/pup-filter.txt: 189-0, 6h.36m
  https://secure.fanboy.co.nz/fanboy-annoyance_ubo.txt: 103212-24266, 12h.38m
  adguard-generic: 84048-4242, 1d.16h.11m
  adguard-mobile: 9813-65, 1d.16h.11m
  adguard-spyware-url: 1709-126, 1d.16h.11m
  block-lan: 64-0, 1d.11h.9m
  curben-phishing: 506-0, 6h.36m
  adguard-social: 23052-1619, 1d.16h.11m
  fanboy-thirdparty_social: 68-19, 1d.16h.11m
  fanboy-cookiemonster: 52139-52137, 1d.20h.12m
  fanboy-social: 17104-17104, 22h.40m
  [3 lists not shown]: [too many]
 default:
  user-filters: 13-0, never
  ublock-filters: 40561-471, 3h.35m Δ
  ublock-badware: 11277-6, 3h.35m Δ
  ublock-privacy: 1255-23, 3h.35m Δ
  ublock-unbreak: 2531-6, 3h.35m Δ
  ublock-quick-fixes: 145-4, 3h.35m Δ
  easylist: 85383-2230, 3h.35m Δ
  easyprivacy: 53082-325, 3h.35m Δ
  urlhaus-1: 25560-0, 6h.36m
  plowe-0: 3726-1589, 223d.14h.37m
filterset (user): [array of 13 redacted]
trustedset:
 added: [array of 50 redacted]
switchRuleset:
 added: [array of 2 redacted]
hostRuleset:
 added: [array of 69 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 3221 ms
 maxAssetCacheWait: 1863 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 27
 network:
  linkedin.com: 2
  licdn.com: 25
 extended:
  ##body:style(overflow: auto !important; position: initial !impor…

```

![image](https://github.com/user-attachments/assets/6525ba92-4630-4c41-9070-f93afebb8c9b)

**Reply by gorhill** [reply, marked as answer]
When I go to `https://www.linkedin.com/`, I see no network requests from `licdn.com` being blocked when using default settings/lists.

Given that you are using heavily customized settings/lists/rules, use [the logger](https://github.com/gorhill/uBlock/wiki/The-logger) to find out what is blocking those network requests. The logger is the first step ro self-diagnosing filtering issues for people who like to customize with custom filters/rules/lists.

**Reply by gerroon** [reply]
Thanks, it seems like this was the culprit list "Block Outsider Intrusion into"

**Reply by gorhill** [reply]
I enabled that list and I am not seeing any issue when navigating to `https://www.linkedin.com/`.

Can you provide more details on how that list is interfering with `https://www.linkedin.com/`? This is a stock list and we need to fix any issue arising from its usage.

**Reply by gerroon** [reply]
This is what I get if I just enable that list again. The site is back if I just uncheck that list.

![image](https://github.com/user-attachments/assets/37c78ef3-f453-4368-b79a-6bc564a70889)

![image](https://github.com/user-attachments/assets/8cfb83a9-ab4b-40a6-898f-0e655d131159)

**Reply by stephenhawk8054** [reply]
Can you do these steps (follow strictly the step order, don't mix the steps orders, or skip any steps):

1. Click on uBO icon > ▤ ["The logger" icon](https://github.com/gorhill/uBlock/wiki/The-logger), it will open a logger window
2. Open new tab and reproduce the issue
3. Switch to the logger window that was opened in step 1, [click on search bar, choose `blocked`](https://github.com/gorhill/uBlock/wiki/The-logger#filtering-the-logger-output)
4. Copy the logger via 📋 [export/copy button at the top right](https://github.com/gorhill/uBlock/wiki/The-logger#export-dialog)
5. Paste the logs to [https://www.logpasta.com](https://www.logpasta.com) and share the link here

**Reply by gerroon** [reply]
It looks like there were some updates to the lists. Today I don't see the whole site is being broken but a lot of media on the site are broken. I can get posts showing media by disabling the "Block Outsider Intrusion into LAN" list.

https://www.logpasta.com/paste/ac8a4362-fe1e-43a1-b9da-e77afaa1796b 

![image](https://github.com/user-attachments/assets/45c2255d-3d19-4d33-b908-975f9dcfa855)

**Reply by stephenhawk8054** [reply]
There are no updates to the list. Are you using proxy or VPN in your browser / machine?

**Reply by gorhill** [reply]
@gerroon If you open a new tab and navigate to `about:networking#dnslookuptool`, enter `media.licdn.com` in the _Domain:_ field, then click _Resolve_, what is the result?

**Reply by gerroon** [reply]
My browser is behind Privoxy and my local DNS resolver is Adguard Home that is running on the same PC tnat runs Privoxy. The site normally works until I enable that list. I am not sure why the resolver test returns an error with or without the list enabled.

![image](https://github.com/user-attachments/assets/fd5fee0f-c0b2-4a56-bda9-f355b74f98e1)

**Reply by gorhill** [reply]
If you open Network pane in dev tools for the webpage, click on an entry which is being blocked, what is the "Address:" being reported?

Example:
![image](https://github.com/user-attachments/assets/1ff792b8-ecbf-41a6-bcde-0833c974b871)
